docker-compose.yml

```diff
@@ -16,8 +16,8 @@ services:
       SONAR_ES_BOOTSTRAP_CHECKS_DISABLE: 'true'
     volumes:
       - type: bind
-        source: ./target/ecocode-java-plugin-1.5.1-SNAPSHOT.jar
-        target: /opt/sonarqube/extensions/plugins/ecocode-java-plugin-1.5.1-SNAPSHOT.jar
+        source: ./target/ecocode-java-plugin-1.5.2-SNAPSHOT.jar
```

Comment:
this modifications are not done like this ... this kind of modification is a part of release process (described here https://github.com/green-code-initiative/ecoCode-common/blob/main/doc/INSTALL.md#howto-create-a-release-core-contributor-rights-needed )
modifications not needed

Reply:
Ok, I'll revert the change.

Reply:
sorry, my bad, as we delete deprecated rules, we have to change minor version. you're rigth.
please update this version to 1.6.0-SNAPHOT please

Reply:
ok, i will change